netdata/netdata
# windows.plugin (IIS)

##### Summary
This PR brings IIS metrics to windows.plugin.

##### Test Plan

1. Install IIS on your host. If you never had to do this before, you can follow [this](https://docs.appeon.com/ps2022r2/Installing_IIS_in_Windows_11.html) instructions.
2. Check if it is running, in theory you will have port 80 open in your host.
3. Compile this branch
4. Build the installer 
5. Install netdata. Access the local server, and you should have metrics:

![Screenshot_20240918_025259](https://github.com/user-attachments/assets/996d8b03-cda0-4113-8b6b-f20cbe1d4eec)


##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
